pythonforandroid/recipes/python3/__init__.py

```diff
@@ -323,12 +330,17 @@ def build_arch(self, arch):
                     *(' '.join(self.configure_args).format(
                                     android_host=env['HOSTARCH'],
                                     android_build=android_build,
+                                    python_host_bin=join(self.get_recipe(
+                                        'host' + self.name, self.ctx
+                                    ).get_path_to_python(), "python3"),
                                     prefix=sys_prefix,
                                     exec_prefix=sys_exec_prefix)).split(' '),
                     _env=env)
 
+            # disable parallel build on 3.11 to fix:
+            # ld: error: unable to find library -lpython3.11
```

Comment:
Is a sort of known issue, or something you encountered during your tests?

Reply:
I encountered this during building, It builds successfully but when compling some extensions it gives this errors:
```
building '_math' extension
ld: error: unable to find library -lpython3.11
clang-14: error: linker command failed with exit code 1 (use -v to see invocation)
```
With this buggy build It gives `ModuleNotFoundError: No module named 'math'` while importing kivy.